gdrive-files-to-microsoft-onedrive-files/main.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/io;
+import ballerina/log;
+import ballerinax/googleapis.drive;
+import ballerinax/microsoft.onedrive;
+
+configurable string gDriveAccessToken = ?;
+configurable string gDriveFolderId = ?;
+
+configurable string oneDriveAccessToken = ?;
+configurable string oneDrivePath = ?;
+
+configurable boolean filesOverridable = ?;
+
+public function main() returns error? {
+    drive:Client gDriveEndpoint = check new ({auth: {token: gDriveAccessToken}});
+    onedrive:Client onedriveEndpoint = check new ({auth: {token: oneDriveAccessToken}});
+
+    io:println("Starting to sync files from Google Drive to Microsoft OneDrive...");
+
+    string gDriveQuery = string `'${gDriveFolderId}' in parents and trashed = false`;
+    stream<drive:File> filesInGDrive = check gDriveEndpoint->getAllFiles(gDriveQuery);
+
+    io:println("Retrieved files from Google Drive...");
```

Comment:
this printline is unnecessary?

Reply:
Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)